docs/pages/guides/schema-validation.mdx

```diff
@@ -48,10 +48,26 @@ the storage that do not match the schema you provided. Situations like this
 should only happen in development mode, and the developer is responsible for
 fixing them.
 
-## Create a schema
+## Schema Operations
```

Comment:
```suggestion
## Schema actions
```

I'd prefer not to use "operations" here, because other things are called operations already.